RNSDK [Android] "Cannot find symbol method getCurrentActivity()"

### Description:

When I try to build an Android React Native app, I see the following error:
```
node_modules/@jitsi/react-native-sdk/android/src/main/java/org/jitsi/meet/sdk/JMOngoingConferenceModule.java:27: error: cannot find symbol
        Context activityContext = context.getCurrentActivity();
                                         ^
  symbol:   method getCurrentActivity()
  location: variable context of type Context
```
on [this line](https://github.com/jitsi/jitsi-meet/blob/master/react-native-sdk/android/src/main/java/org/jitsi/meet/sdk/JMOngoingConferenceModule.java#L27)

### Steps to reproduce:

1. Integrate `@jitsi/react-native-sdk` to react-native project
2. Try to build it for Android

### Expected behavior:

It should build

### Actual behavior:

I receive an error `Cannot find symbol method getCurrentActivity()`

### Client information:

- Operating System: Android
- React Native: 0.72.3
- Jitsi React Native SDK: 1.0.0
- Android Gradle config:
```
        buildToolsVersion = "33.0.0"
        minSdkVersion = 24
        compileSdkVersion = 33
        targetSdkVersion = 33
        supportLibVersion = "30.0.0"

        ndkVersion = "23.1.7779620"
        kotlinVersion = "1.8.21"
```

### Additional information:

I switched from `@jitsi/react-native-sdk` v0.5.0 to v1.0.0, and I got this error

--- a/react-native-sdk/android/src/main/java/org/jitsi/meet/sdk/JMOngoingConferenceModule.java
+++ b/react-native-sdk/android/src/main/java/org/jitsi/meet/sdk/JMOngoingConferenceModule.java
@@ -1,6 +1,6 @@
 package org.jitsi.meet.sdk;
 
-import android.app.Notification;
+import android.app.Activity;
 import android.content.Context;
 
 import androidx.annotation.NonNull;
@@ -24,9 +24,9 @@ class JMOngoingConferenceModule
     @ReactMethod
     public void launch() {
         Context context = getReactApplicationContext();
-        Context activityContext = context.getCurrentActivity();
+        Activity currentActivity = getCurrentActivity();
 
-        JitsiMeetOngoingConferenceService.launch(context, activityContext);
+        JitsiMeetOngoingConferenceService.launch(context, currentActivity);
     }
 
     @ReactMethod

--- a/react-native-sdk/android/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java
+++ b/react-native-sdk/android/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java
@@ -16,6 +16,7 @@
 
 package org.jitsi.meet.sdk;
 
+import android.app.Activity;
 import android.app.Notification;
 import android.app.NotificationManager;
 import android.app.Service;
@@ -41,9 +42,9 @@ import java.util.HashMap;
 public class JitsiMeetOngoingConferenceService extends Service {
     private static final String TAG = JitsiMeetOngoingConferenceService.class.getSimpleName();
 
-    public static void launch(Context context, Context activityContext) {
+    public static void launch(Context context, Activity currentActivity) {
 
-        RNOngoingNotification.createOngoingConferenceNotificationChannel(activityContext);
+        RNOngoingNotification.createOngoingConferenceNotificationChannel(currentActivity);
 
         Intent intent = new Intent(context, JitsiMeetOngoingConferenceService.class);
 

--- a/react-native-sdk/android/src/main/java/org/jitsi/meet/sdk/RNOngoingNotification.java
+++ b/react-native-sdk/android/src/main/java/org/jitsi/meet/sdk/RNOngoingNotification.java
@@ -16,6 +16,7 @@
 
 package org.jitsi.meet.sdk;
 
+import android.app.Activity;
 import android.app.Notification;
 import android.app.NotificationChannel;
 import android.app.NotificationManager;
@@ -41,19 +42,19 @@ class RNOngoingNotification {
 
     static final int NOTIFICATION_ID = new Random().nextInt(99999) + 10000;
 
-    static void createOngoingConferenceNotificationChannel(Context activityContext) {
+    static void createOngoingConferenceNotificationChannel(Activity currentActivity) {
 
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
             return;
         }
 
-        if (activityContext == null) {
+        if (currentActivity == null) {
             JitsiMeetLogger.w(TAG + " Cannot create notification channel: no current context");
             return;
         }
 
         NotificationManager notificationManager
-            = (NotificationManager) activityContext.getSystemService(Context.NOTIFICATION_SERVICE);
+            = (NotificationManager) currentActivity.getSystemService(Context.NOTIFICATION_SERVICE);
 
         NotificationChannel channel
             = notificationManager.getNotificationChannel("JitsiOngoingConferenceChannel");
@@ -62,7 +63,7 @@ class RNOngoingNotification {
             return;
         }
 
-        channel = new NotificationChannel("JitsiOngoingConferenceChannel", activityContext.getString(R.string.ongoing_notification_channel_name), NotificationManager.IMPORTANCE_DEFAULT);
+        channel = new NotificationChannel("JitsiOngoingConferenceChannel", currentActivity.getString(R.string.ongoing_notification_channel_name), NotificationManager.IMPORTANCE_DEFAULT);
         channel.enableLights(false);
         channel.enableVibration(false);
         channel.setShowBadge(false);

